buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.sh

```diff
@@ -266,6 +266,9 @@ check () {
     make/hotspot/*)
       trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion make/hotspot/*)"
       IN_JDK=0;;
+    make/data/hotspot-symbols/*)
+      trace  "$1 deemed not to be in the built JDK because it matches case parameter expansion make/hotspot/*)"
```

Comment:
Pls fix the trace message to contain `make/data/hotspot-symbols`

Reply:
@pshipton thanks, fixed now. Pull request is updated.